### PR TITLE
Adjust change since Melting Pot 2.0

### PR DESCRIPTION
**Pull request by ninell-oldenburg**
This at least made it work for me, I was thinking it was somehow connected to the introduction of roles in 2.0

### CODE REVIEW

File: examples/rllib/view_models.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def main():
   # Create a new environment to visualise
   env = utils.env_creator(config["env_config"]).get_dmlab2d_env()
 
-  num_bots = config["env_config"]["num_players"]
+  num_bots = len(config["env_config"]["roles"])
```
Comment (duenez) [inline examples/rllib/view_models.py:63]: This is not quite right... The default number of players should be inferred from the length og the `default_player_roles` in the config.

Comment (ninell-oldenburg) [reply examples/rllib/view_models.py:63]: Interesting! I'm afraid I cannot find the config where the `default_player_roles` shall be in.

Comment (duenez) [reply examples/rllib/view_models.py:63]: They are in the substrate configs, e.g.:
https://github.com/deepmind/meltingpot/blob/ad1cead1e77138d1f20545f508d6bbc459746582/meltingpot/python/configs/substrates/allelopathic_harvest__open.py#L81

Comment (ninell-oldenburg) [reply examples/rllib/view_models.py:63]: Ah got it, thank you!

**Comment by ninell-oldenburg** [comment]
Sure, thanks for the reminder :)

**Review by duenez — APPROVED** [review]

**Comment by duenez** [comment]
I believe this is already at head? I'm closing the PR, but feel free to re-open if I got it wrong.

**Pull request closed without merging**